lighthouse-core/report/html/renderer/util.js

```diff
@@ -373,6 +373,10 @@ class Util {
       return value;
     }
 
+    if (!/:\/\//.test(value)) {
+      value = 'http://' + value;
+    }
+
```

Comment:
Well, this might be too much of a generic thing here. Should probably move closer to my change. What do you think?